tests/Agent/IntegrationTests/IntegrationTests/IntegrationTests.csproj

```diff
@@ -27,6 +27,14 @@
   <ItemGroup>
     <None Remove="RejitLogLines.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="para-xunit.runner.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="xunit.runner.json">
```

Comment:
Having the xunit.runner.json (default with parallel disabled) should affect any usage of xunit since it will look for the json in the same folder as the test dll.  Are you suggesting something like instead of having the default json being configured with parallel disabled, we just have the schema in there and use scripting to add the bits for CI?  That way our VS tests will be unaffected but CI will still get the right options?

Reply:
With the change to dynamically creating the json files in CI, the local test runs should be unaffected now

Reply:
Nice!